src/sharepoint/rest/web/lists/contenttypes/contenttypes.ts

```diff
@@ -9,7 +9,7 @@ export class ContentTypes extends Queryable {
         super(url, "/ContentTypes");
     }
     public getById(id: string) {
-        this._url.push(`(${id})`);
+        this._url.push(`(\"${id}\")`);
```

Comment:
Test required that we got "GUID", not just GUID which this returned before.
The actual way I did the string interpolation here to get "GUID" seems a bit wonky tho. Better ideas?